docs/index.asciidoc

```diff
@@ -160,8 +160,8 @@ For example, if we have configured `fallback => "no match"`, using this dictiona
 [source,ruby]
     foo: bar
 
-Then, if logstash received an event with the field `foo` set to `bar`, the destination
-field would be set to `bar`. However, if logstash received an event with `foo` set to `nope`,
+Then, if logstash received an event with the field `data` set to `foo`, the destination
```

Comment:
The current documentation is correct: with the fallback dictionary `foo: bar`, it means that if the field `foo` has no translation, its value will be `bar`. There is no field called `data` as this change implies